src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx

```diff
@@ -107,6 +107,16 @@ To instrument [`Transactions`](/docs/apm/applications-menu/monitoring/transactio
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Adds user tracking to the current transaction by setting the `enduser.id` agent attribute.
```

Comment:
The wording for this should probably be patterned after what's already there for consistency, eg:

"Add user tracking to Transactions"

And the "user tracking" text would then link out

Reply:
Since I do not have permissions to push as of now, the phrasing would be:

Adds [user tracking](/docs/apm/agents/java-agent/attributes/java-agent-attributes#user-attributes) to `Transactions` by setting the `enduser.id` agent attribute.

Reply:
I'll take care of these.

Reply:
Done

Reply:
Thanks!